packages/create-cloudflare/templates-experimental/next/c3.ts

```diff
@@ -17,16 +19,39 @@ const generate = async (ctx: C3Context) => {
 	]);
 };
 
-const configure = async () => {
+const configure = async (ctx: C3Context) => {
 	const packages = [
-		"@opennextjs/cloudflare@0.3.x",
+		"@opennextjs/cloudflare@0.4.x",
 		"@cloudflare/workers-types",
 	];
 	await installPackages(packages, {
 		dev: true,
 		startText: "Adding the Cloudflare adapter",
 		doneText: `${brandColor(`installed`)} ${dim(packages.join(", "))}`,
 	});
+
+	const usesTs = usesTypescript(ctx);
+	updateNextConfig(usesTs);
+};
+
+const updateNextConfig = (usesTs: boolean) => {
+	const s = spinner();
+
+	const configFile = `next.config.${usesTs ? "ts" : "mjs"}`;
+	s.start(`Updating \`${configFile}\``);
+
+	const configContent = readFile(configFile);
+
+	const updatedConfigFile =
+		`import { initOpenNextCloudflareForDev } from '@opennextjs/cloudflare';
+
+		initOpenNextCloudflareForDev();
+
+		`.replace(/\n\t*/g, "\n") + configContent;
```

Comment:
nits: maybe move this to the end (to leave the Next config on top), add a comment?

```suggestion
	const updatedConfigFile =  configContent +
		`
		// added by create cloudflare to enable calling `getCloudflareContext()` in `next dev`
		import { initOpenNextCloudflareForDev } from '@opennextjs/cloudflare';
		initOpenNextCloudflareForDev();
		`.replace(/\n\t*/g, "\n");
```